alert/store.go

```diff
@@ -481,7 +481,7 @@ func (s *Store) Create(ctx context.Context, a *Alert) (*Alert, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer tx.Rollback()
+	defer sqlutil.Rollback(ctx, "Create", tx)
```

Comment:
Seeing "Create" in the logs is going to be challenging to troubleshoot by itself, add the package and describe the action "alert: create" or "create alert"

Reply:
sure thing.  went with alert: create